golem_messages/cryptography.py

```diff
@@ -317,5 +325,6 @@ def sign(self, data):
         return signature
 
     def verify(self, signature, message):
-        assert len(signature) == 65
+        if len(signature) != 65:
+            raise exceptions.INvalidSignature('Invalid length')
```

Comment:
Shouldn't be ` raise exceptions.InvalidSignature('Invalid length')'`  instead of `raise exceptions.INvalidSignature('Invalid length')`?

Reply:
Already fixed in 9c30537 :)